eth_tester/utils/address.py

```diff
@@ -2,8 +2,9 @@
 
 from eth_utils import (
     keccak,
+    to_canonical_address,
 )
 
 
 def generate_contract_address(address, nonce):
-    return keccak(rlp.encode([address, nonce]))[-20:]
+    return to_canonical_address(keccak(rlp.encode([address, nonce]))[-20:])
```

Comment:
What do you think about breaking this up into a few lines?

Something like:
```py
next_account_hash = keccak(rlp.encode([address, nonce]))
return to_canonical_address(next_account_hash[-20:])

Reply:
Done, thanks.